docs/book/src/topics/specify-management-iam-role.md

```diff
@@ -0,0 +1,47 @@
+# Specifying the IAM Role to use for Management Components
+
+## Prerequisites
```

Comment:
This document needs to be separated out into seperate instructions for IRSA and Kiam. The latter is still heavily used.